plenum/test/instances/conftest.py

```diff
@@ -0,0 +1,2 @@
+from plenum.test.pool_transactions.conftest import looper
```

Comment:
I don't think it is a good practice to use conftest for imports only.
move imports to files where they are needed and remove this conftest if it is empty